UnitTest.Rollbar/RollbarLoggerFixture.cs

```diff
@@ -768,7 +768,7 @@ private void Logger_InternalEvent(object sender, RollbarEventArgs e)
         /// Defines the test method MultithreadedStressTest_BlockingLogs.
         /// </summary>
         [TestMethod]
-        [Timeout(80000)]
+        [Timeout(120000)]
```

Comment:
Why are we increasing the timeout period?

Reply:
These integration tests were timing out since they needed 85000 to 90000 msecs to complete. bumped the timeout higher to prevent the unit-tests from false timeouts when running on slower/busy hosts.